### PR TITLE
Assert that ASEditableTextNode stays view-backed

### PR DESCRIPTION
**Pull request by adly-holler**

### CODE REVIEW

File: AsyncDisplayKit/ASEditableTextNode.mm [hunk 1]
```diff
@@ -188,6 +188,11 @@ - (void)setOpaque:(BOOL)opaque
   _placeholderTextKitComponents.textView.opaque = opaque;
 }
 
+- (void)setLayerBacked:(BOOL)layerBacked {
```
Comment (appleguy) [inline AsyncDisplayKit/ASEditableTextNode.mm]: Open bracket on newline.

Expand the comment below, by revising after the dash - "instances must be view-backed in order to ensure touch events can be passed to the internal UITextView during editing"

File: AsyncDisplayKit/ASEditableTextNode.h [hunk 2]
```diff
@@ -11,7 +11,7 @@
 
 @protocol ASEditableTextNodeDelegate;
 
-/// @abstract ASEditableTextNode implements a node that supports text editing.
+/// @abstract ASEditableTextNode implements a view-backed node that supports text editing.
```
Comment (appleguy) [inline AsyncDisplayKit/ASEditableTextNode.h]: Optional, but might be better to say as a separate sentence "Does not support layer-backing".

**Comment by appleguy** [comment]
Thanks @adly-holler!  Small stylistic change here, ready to merge after.

**Comment by adly-holler** [comment]
Changes made!

**Pull request merged**